### PR TITLE
fix(minor): some ui assets problems

### PR DESCRIPTION
**Pull request by g1eny0ung**
Signed-off-by: Yue Yang <g1enyy0ung@gmail.com>

### What is changed and how does it work?

- Remove `.DS_Store` if it exists before embed UI assets
- Correct a filename

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Review by cwen0 — APPROVED** [review]
LGTM

File: pkg/apiserver/server.go [hunk 1]
```diff
@@ -102,7 +102,7 @@ func newDashboardRouter(r *gin.Engine, ui http.FileSystem) {
 		c.FileFromFS(c.Request.URL.Path, ui)
 	}
 
-	r.GET("dashboard/*any", func(c *gin.Context) {
+	r.GET("/dashboard/*any", func(c *gin.Context) {
```
Comment (cwen0) [inline pkg/apiserver/server.go:105]: Could this issue cause any anomalies or API errors?

Comment (g1eny0ung) [reply pkg/apiserver/server.go:105]: No, this is just a unified change, prepend a slash to all defined routes. Add it or not won't influence the real effect.

**Review by fewdan — APPROVED** [review]
LGTM

**Pull request merged**